src/main/java/org/profit/telegrambot/util/KeyboardUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.profit.telegrambot.util;
+
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class KeyboardUtil {
+    public static ReplyKeyboardMarkup contactMarkup() {
+        KeyboardButton contactButton = new KeyboardButton("Send number");
+        contactButton.setRequestContact(true);
+        KeyboardRow row = getRow(contactButton);
+        List<KeyboardRow> rowList = getRowList(row);
+
+        return getMarkup(rowList);
+    }
+    public static ReplyKeyboardMarkup adminMenu() {
+        KeyboardButton button = getButton("Category CRUD");
+        KeyboardButton button1 = getButton("Product CRUD");
+        KeyboardButton button2 = getButton("Announcement");
+
+        KeyboardRow keyboardRow = getRow(button, button1, button2);
+
+        return getMarkup(Collections.singletonList(keyboardRow));
+    }
+    private static KeyboardButton getButton(String demo) {
+        return new KeyboardButton(demo);
+    }
+
+    private static KeyboardRow getRow(KeyboardButton ... buttons) {
+        return new KeyboardRow(Arrays.asList(buttons));
+    }
+
+    private static List<KeyboardRow> getRowList(KeyboardRow ... rows) {
+        return Arrays.asList(rows);
+    }
+
+    private static ReplyKeyboardMarkup getMarkup(List<KeyboardRow> keyboard) {
```

Comment:
Seems the method does not **get**s a markup. It **create**s it. 

The **get** methods by convention (rather by implicit convention, i.e. it is common to do so) return a state of an instance/class. In contrast, this function returns a new instance each call.